test/samples/indentation.clj

```diff
@@ -228,15 +228,25 @@
  :foo
  "bar"}
 
-;; NOTE: List elements with metadata are not indented correctly.
+;; NOTE: It works well now with the alternative grammar.
```

Comment:
I'm not sure this wording makes sense without the former context, so perhaps it can just be removed or changed to "requires the unstable grammar to work properly".